kopeme-core/src/main/java/de/dagere/kopeme/datastorage/FolderProvider.java

```diff
@@ -12,21 +12,24 @@ public class FolderProvider {
 
    static final Long MEASURE_TIME = Long.valueOf(System.currentTimeMillis());
 
-   static final String KOPEME_DEFAULT_FOLDER = System.getenv("KOPEME_HOME") != null ? System.getenv("KOPEME_HOME")
-         : AndroidConfiguration.read("KOPEME_HOME") != null ? AndroidConfiguration.read("KOPEME_HOME")
-         : System.getenv("HOME") + File.separator + ".KoPeMe" + File.separator;
```

Comment:
The static final String caused errors in `FolderProviderTest` and `AndroidConfigurationTest`. 
Whichever test used `FolderProvider.getInstance()` first would set the string and it would carry over to the other test and cause a failure.